subgraph.yaml

```diff
@@ -0,0 +1,90 @@
+specVersion: 1.2.0
+indexerHints:
+  prune: auto
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: PositionRegistry
+    network: optimism
```

Comment:
how `build --network <n>` is working I was assuming we would have var/template values for networks and addresses in the `subgraph.yaml`

Reply:
if we go via template then we will use handlebar or similar library to generate `subgraph.yaml`. Current setup is default and it will update existing subgraph.yaml with values from network provided at build time. Once yaml is update, user can proceed to deployment.

Reply:
got it, thanks for clarifying it